glutin/src/api/wgl/mod.rs

```diff
@@ -705,6 +712,8 @@ unsafe fn choose_arb_pixel_format(
             get_info(gl::wgl_extra::FRAMEBUFFER_SRGB_CAPABLE_ARB) != 0
         } else if extensions.split(' ').find(|&i| i == "WGL_EXT_framebuffer_sRGB").is_some() {
             get_info(gl::wgl_extra::FRAMEBUFFER_SRGB_CAPABLE_EXT) != 0
+        } else if extensions.split(' ').find(|&i| i == "WGL_EXT_colorspace").is_some() {
+            get_info(gl::wgl_extra::FRAMEBUFFER_SRGB_CAPABLE_EXT) != 0
```

Comment:
Unlike `egl`, I'm guessing the window creation logic does not need to change at all for `wgl`. Correct?

Reply:
Yes, `wgl` uses the pixel format and `egl` needs changes to `CreateWindowSurface`.